src/Table.js

```diff
@@ -44,16 +46,17 @@ const Table = (props) => {
     size ? 'table-' + size : false,
     bordered ? 'table-bordered' : false,
     striped ? 'table-striped' : false,
-    inverse ? 'table-inverse' : false,
+    (dark || inverse) ? 'table-dark' : false,
     hover ? 'table-hover' : false,
     reflow ? 'table-reflow' : false
```

Comment:
Just noticing reflow was removed in bootstrap alpha.6, can you kill it in this PR :pray: